src/main/java/org/testng/internal/RuntimeBehavior.java

```diff
@@ -16,4 +18,16 @@ public static boolean isDryRun() {
         String value = System.getProperty("testng.mode.dryrun", "false");
         return Boolean.parseBoolean(value);
     }
+
+    /**
+     * @return - returns the {@link TimeZone} corresponding to the JVM argument
+     * <code>-Dtestng.timezone</code> if it was set. If not set, it returns the default
+     * timezone pertaining to the user property <code>user.timezone</code>
+     */
+    public static TimeZone getTimeZone() {
+        if (System.getProperty("testng.timezone", "").trim().isEmpty()) {
+            return TimeZone.getDefault();
+        }
+        return TimeZone.getTimeZone(System.getProperty("testng.timezone"));
```

Comment:
Maybe extract `System.getProperty("testng.timezone", "")` in a variable to avoid the repetition?

Reply:
Fixed, amended commit and pushed the changes